meta-balena-common/recipes-core/initrdscripts/files/cryptsetup-efi-tpm

```diff
@@ -33,25 +66,25 @@ cryptsetup_enabled() {
         return 1
     fi
 
+    return 0
+}
+
+cryptsetup_run() {
```

Comment:
We'll probably need to change this from always running, to always running when the device is in user mode. Otherwise, systems without secure boot may not boot.